tests/test_model.py

```diff
@@ -177,6 +177,11 @@ def test_device_or_providers(device):
             None,
             pytest.FEATURE,
         ),
+        (
+            pytest.MODEL_DYNAMIC_PATH,
+            pytest.FEATURE.feature_names,
+            pytest.FEATURE,
+        ),
```

Comment:
Do we have tests for the following model dimensions?

* `(1, 1)`
* `(1, 1, -1)`
* `(1, 1, 1)`
* `(1, 2, -1)`
* `(1, 2, 1)`
* `(2, 2, -1)`
* `(2, 2, 2)`

Unfortunately, this is not so easy to understand from the test definitions. There we have `pytest.MODEL_SINGLE_PATH`, `pytest.MODEL_SINGLE_PATH`, `pytest.MODEL_MULTI_PATH`, but even when looking in `tests/conftest.py` it is not obvious to me, what the expected output dimensions of those models are.

Reply:
You are right, we need a cleaner way to create our test models.

Reply:
I think I found a solution, but prefer to work on it in a separate PR. Created https://github.com/audeering/audonnx/issues/22

Reply:
OK, then for now I simply assume that tests are sufficient and will complain later on to add more tests when it is easier.